python/activator/visit.py

```diff
@@ -1,3 +1,5 @@
+__all__ = ["Visit"]
```

Comment:
No license statement?

Reply:
I didn't want to muck with the visit.py files, because we may want to consolidate those somehow (there's two copies in different places).

Reply:
This is the real one, the other is a symlink. I'm pretty sure we can delete the latter if we change the import in `upload.py`, I guess I was waiting for test coverage of `upload` before trying. 🙂

Reply:
Fair point, but I'm nervous about writing tests of `upload` (see KT's comment on DM-33938: `upload.py` is a test itself)

Reply:
I disagree -- for now, at least, `upload.py` is the server for which `activator.py` is the client; that makes it part of the system we're developing. In particular, we probably will need to make changes to `upload.py` as we start doing more substantial integration testing.